bigtable-client-core-parent/bigtable-hbase/pom.xml

```diff
@@ -168,6 +168,12 @@ limitations under the License.
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>net.bytebuddy</groupId>
+      <artifactId>byte-buddy</artifactId>
+      <version>1.12.6</version>
```

Comment:
nit - I think this is used now in more than one place? Can we extract the version number to a property?